docs/blog/2019-03-11-dot-org-prototypes/index.md

```diff
@@ -0,0 +1,139 @@
+- Debate over how to help developers know that Gatsby also has some upcoming commercial offerings. What do we call the commercial offerings so they make sense to first time visitors?
+
+### Settled concerns:
+
+"Install Gatsby" is an more specific call to action, so better than “Get Started” probably.
+
+> Spoiler alert! The button label “Install Gatsby” did not work in the usability tests. Nearly everyone thought that when they clicked on it, they would be taken to their phone’s app store.
+
+## What is the final lo-fi prototype?
+
+Here is the [final lo-fi prototype](https://www.figma.com/proto/0rROnAzH4mFLYLEx9TDleN/For-Flo---Gatsbyjs.org-mobile-homepage-prototype?node-id=140%3A487&viewport=-1603%2C423%2C1&scaling=scale-down) that we’ll use to guide the next phase of visual design. Below, I list the reasons we’re moving forward with this prototype.
+
+### What is Gatsby?
+
+The definition was clear in the prototype and the testers could explain what Gatsby is after glancing at the prototype for only a few seconds. This was a success!
+
+### Why use Gatsby?
+
+We learned that Lighthouse scores weren’t as persuasive to testers as we anticipated because many people don’t know how the scores are calculated. Other ideas that could show Gatsby’s value more concretely might be case studies, sites in the site showcase, a diagram of how Gatsby works, code snippets, and more.
+
+### What can I use Gatsby for?
+
+This is a question that testers had and the prototype didn’t quite answer it, so we concluded that we will simply answer that question above the fold on the homepage and then include a button labelled “Learn more” that leads to a list of the kinds of sites people build with Gatsby and resources for people who want to build each kind of site.
+
+### How do I use Gatsby?
+
+The “Install Gatsby” button label did not work in the usability tests. Nearly everyone thought that when they clicked on it, they would be taken to their phone’s app store. For this reason, we are sticking with “Get started” as a button label (this worked in usability tests).
+
+## What we've done with the research so far
+
+We made two changes to fix the initial problems uncovered by usability testing. The "How it Works" diagram was moved up so some is visible above the fold, since that diagram was easier to quickly understand than the text that now is below the diagram. Also, the subheading under "How it Works" is changed to make it clear that Gatsby is open source and free, and make it clear that it's a tool for developers.
+
+![Screenshot of gatsbyjs.org homepage on a mobile device](mobile-prototype-2019-03-01.png)
+[Move diagram above card and change explanatory text](https://github.com/gatsbyjs/gatsby/pull/12236)
+
+And then refined those changes by changing the headline and subheadline.
```

Comment:
This should be a complete sentence: "Then we refined those changes by modifying..."